src/clients/SpokePoolClient.ts

```diff
@@ -420,15 +420,17 @@ export class SpokePoolClient {
       toBlock: this.eventSearchConfig.toBlock || this.latestBlockNumber,
       maxBlockLookBack: this.eventSearchConfig.maxBlockLookBack,
     };
+    if (searchConfig.fromBlock > searchConfig.toBlock) {
+      this.log("warn", `Invalid update() searchConfig.`, searchConfig);
```

Comment:
Fix use of this.log():

```suggestion
      this.log("warn", `Invalid update() searchConfig.`, { searchConfig });
```